spec/requests/organisations_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'rails_helper'
+
+RSpec.describe "/organisations", type: :request do
+  fixtures :all
+
+  let(:valid_attributes) do
+    { name: 'Foo'  }
+  end
+
+  let(:invalid_attributes) do
+    { name: nil }
+  end
+
+  let(:user) { users :admin }
+
+  before { sign_in user }
+
+  describe "GET /index" do
+    it "renders a successful response" do
+      Organisation.create! valid_attributes
```

Comment:
This test is mainly as created via scaffolding. All these `Organisation.create!` calls are a little clunky but will do for now.